components/tidb_query/src/util.rs

```diff
@@ -5,6 +5,41 @@ use tipb::ColumnInfo;
 
 use crate::codec::datum::Datum;
 
+const MAX_RAND_VALUE: u64 = 0x3FFFFFFF;
+
+pub struct MySQLRng {
+    seed1: u32,
+    seed2: u32,
+}
+
+impl MySQLRng {
+    fn from_seeds(seed1: u32, seed2: u32) -> Self {
+        MySQLRng {
+            seed1: seed1 % MAX_RAND_VALUE as u32,
+            seed2: seed2 % MAX_RAND_VALUE as u32,
+        }
+    }
```

Comment:
Can this fn be merged with `new_with_seed` since it's only called once

Reply:
removed.